src/plugins/condition/components/Condition.vue

```diff
@@ -285,10 +280,13 @@ export default {
             return this.currentCriteria && identifier;
         },
         persist() {
-            this.openmct.objects.mutate(this.domainObject, 'configuration', this.domainObject.configuration);
+            this.$emit('updateCondition', {
+                condition: this.domainObject,
+                index: this.conditionIndex
+            });
         },
-        initCap: function (string) {
-            return string.charAt(0).toUpperCase() + string.slice(1)
+        initCap: function (sentence) {
```

Comment:
Is this because string is a reserved word? It seems "word" or "phrase" would make more sense than "sentence" in this context, since none of the existing options are sentences.

Reply:
Right. Let me know what word you would prefer and I'll change it.